### PR TITLE
Beta Notice

### PR DESCRIPTION
**Pull request by MJ1227UB**
**New beta badge!**

This PR will add the following

- A new type on Watt Badge: "Beta"
- A new attribute on Watt Tooltip: "Variant" - to choose between dark or light version of the tooltip
   - (And a control in the storybook to showcase it 🙂)
- A beta notice with a tooltip on the sidenav of EO
- A beta notice on the Footer of both EO and EO Landing page
- Altered command in translation readme to use "bun" instead of "yarn"

Here's some before and after screenshots:

**Nav bar:**
Before:
Screenshot 2024-11-12 150427
After:
Screenshot 2024-11-12 150403

**Footer:**
Before:
Screenshot 2024-11-12 150432
After:
Screenshot 2024-11-12 150416

### CODE REVIEW

File: libs/eo/core/shell/src/lib/eo-shell.component.ts [hunk 1]
```diff
@@ -87,6 +91,13 @@ import { EoAccountMenuComponent } from './eo-account-menu';
         width: 80%;
       }
 
+      .beta-badge {
+        display: flex;
```
Comment (frontend-specialisten) [inline libs/eo/core/shell/src/lib/eo-shell.component.ts:95]: Is this necessary?

**Review by frontend-specialisten — COMMENTED** [review]
Great work! just had some minor comments / considerations :)

File: libs/eo/core/shell/src/lib/eo-shell.component.ts [hunk 2]
```diff
@@ -87,6 +91,13 @@ import { EoAccountMenuComponent } from './eo-account-menu';
...
+        justify-self: center;
```
[the remark this answers is not in the document]
Comment (MJ1227UB) [reply libs/eo/core/shell/src/lib/eo-shell.component.ts:96]: It was to center the badge 🙂 
image
Do you prefer another way?

File: libs/eo/shared/components/ui-footer/footer.component.ts [hunk 3]
```diff
@@ -87,6 +88,13 @@ const selector = 'eo-footer';
       .logo {
         width: 264px;
         grid-area: logo;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      }
+
+      .beta-badge-margin {
+        margin-top: 12px;
```
[the remark this answers is not in the document]
Comment (MJ1227UB) [reply libs/eo/shared/components/ui-footer/footer.component.ts:97]: Yes! 🙌

File: libs/eo/shared/components/ui-footer/footer.component.ts [hunk 4]
```diff
@@ -141,7 +149,10 @@ const selector = 'eo-footer';
   `,
   template: `
     <footer>
-      <img eoProductLogo version="secondary" class="logo" />
+      <div class="logo">
+        <img eoProductLogo version="secondary" />
+        <watt-badge type="beta" class="beta-badge-margin">Beta</watt-badge>
```
[the remark this answers is not in the document]
Comment (MJ1227UB) [reply libs/eo/shared/components/ui-footer/footer.component.ts:154]: Good idea! 👍

**Review by frontend-specialisten — APPROVED** [review]

**Pull request merged**